pipeline-model-definition/src/test/resources/errors/topLevelStageGroupsDeepNesting.groovy

```diff
@@ -0,0 +1,52 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2017, CloudBees, Inc.
```

Comment:
It's a new year, a new life, a new dawn...

Reply:
Ah, but it wasn't when I created that file. =)